parquet-thrift/src/main/java/org/apache/parquet/thrift/struct/CompatibilityChecker.java

```diff
@@ -32,7 +34,7 @@
 import org.apache.parquet.thrift.struct.ThriftType.StringType;
 
 /**
- * A checker for thrift struct, returns compatibility report based on following rules:
+ * A checker for thrift struct to enforce its backward compatibility, returns compatibility report based on following rules:
  * 1. Should not add new REQUIRED field in new thrift struct. Adding optional field is OK
  * 2. Should not change field type for an existing field
  * 3. Should not delete existing field
```

Comment:
We consider deleting a field as an incompatible change. This is more of a internal requirement for us. For example when you move a field from one struct to another one, it requires data consumer to explicitly deal with the situation to avoid data loss and we don't want that to happen